common/static/common/js/common_libraries.js

```diff
@@ -1 +1 @@
-define(['domReady!', 'jquery', 'backbone', 'underscore', 'gettext']);
+define(['domReady!', 'jquery', 'jquery-migrate', 'backbone', 'underscore', 'gettext']);
```

Comment:
You're right. I tried to revert this change in a fixup commit but it must have not been applied. I'll fix it.